### PR TITLE
Fix components descriptions

### PR DESCRIPTION
**Pull request by EVDW**

### CODE REVIEW

File: _includes/_components/doctrine-events.html [hunk 1]
```diff
@@ -2,10 +2,11 @@
     <header>
         <div class="container">
             <h2>Doctrine Events</h2>
-            <p>Access the easy way to your doctrine events from the Symfony DIC.</p>
+            <p>A single events doctrine dispatcher in the Symfony dispatcher</p>
```
Comment (akovalyov) [inline _includes/_components/doctrine-events.html]: `A single doctrine events dispatcher in the Symfony one.` ?
`Doctrine to Symfony events redispatcher`?

File: _includes/_components/doctrine-events.html [hunk 2]
```diff
@@ -2,10 +2,11 @@
...
         </div>
     </header>
     <div class="container content">
+        <p>It is able to access the easy way to your doctrine events from the Symfony DependencyInjection Component.</p>
```
Comment (akovalyov) [inline _includes/_components/doctrine-events.html]: `Access your doctrine events from Symfony DependencyInjection component easily` ?

File: _includes/_components/domain-event.html [hunk 3]
```diff
@@ -2,10 +2,11 @@
     <header>
         <div class="container">
             <h2>Domain Event</h2>
-            <p>The raise method allows you to trigger any event in your entity and a Knp\Rad\DomainEvent\Event once the entity is flushed.</p>
+            <p>A lightweight domain event pattern implementation for Doctrine2</p>
         </div>
     </header>
     <div class="container content">
+        <p>The raise method allows you to trigger any event in your entity which will be transform to a Knp\Rad\DomainEvent\Event object and dispatched once the entity has been flushed.</p>
```
Comment (akovalyov) [inline _includes/_components/domain-event.html]: `will be transformed`

File: _includes/_components/user.html [hunk 4]
```diff
@@ -6,6 +6,8 @@
         </div>
     </header>
     <div class="container content">
+        <p>Before your entity is inserted or updated into your database, according traits you choose to use, the salt, plain password or password will be automaticly generated. And all of this is done through three interfaces and three listeners.
```
Comment (PedroTroller) [inline _includes/_components/user.html]:
``` html
<p>yolo</p
```

or

``` html
<p>
    yolo
</p>
```

You have to choose.

**Pull request merged**